### PR TITLE
ruby-tmuxinator: update to 3.2.1 and let it work with tmux 3.4

### PR DESCRIPTION
**Pull request by dataCobra**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64-musl
  - i686

Also adopting the package.

### CODE REVIEW

File: srcpkgs/ruby-tmuxinator/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Thank you for installing tmuxinator.
```
Comment (classabbyamp) [inline srcpkgs/ruby-tmuxinator/INSTALL.msg:1]: this is unnecessary and noisy

**Review by ahesford — CHANGES_REQUESTED** [review]
Drop the `INSTALL.msg` entirely. We don't use messages to teach people how to use the packages that they install.

**Review by ahesford — CHANGES_REQUESTED** [review]
Also, ping the maintainer of the package you intend to adopt, who is still an active Void contributor.

**Comment by dataCobra** [comment]
@egorenar are you fine with me adopting?

File: srcpkgs/ruby-tmuxinator/INSTALL.msg (hunk 1 above)
Comment (dataCobra) [reply srcpkgs/ruby-tmuxinator/INSTALL.msg]: Dropped the INSTALL.msg file.

**Comment by dataCobra** [comment]
As requested, I have removed the INSTALL.msg file.

I've pinged the current maintainer also and he approved my adoption.

**Comment by dataCobra** [comment]
Updated PR from 3.1.2 to 3.2.0.

File: srcpkgs/ruby-tmuxinator/patches/xdg.patch
Comment (classabbyamp) [inline srcpkgs/ruby-tmuxinator/patches/xdg.patch:1]: there's a bunch of changes in this patch that feel unnecessary. can you clean up the diff so it's easier to see what changed?

Comment (dataCobra) [reply srcpkgs/ruby-tmuxinator/patches/xdg.patch:1]: I assume you mean the XDG config changes?

Comment (classabbyamp) [reply srcpkgs/ruby-tmuxinator/patches/xdg.patch:1]: the file i commented on, yes

Comment (dataCobra) [reply srcpkgs/ruby-tmuxinator/patches/xdg.patch:1]: The file should be good to go now.

Please review a last time.

**Pull request merged**